src/applications/mhv/medications/containers/Prescriptions.jsx

```diff
@@ -306,24 +306,11 @@ const Prescriptions = props => {
 
   const handleFullListDownload = async format => {
     updateLoadingStatus(true, 'Downloading your file...');
-    setFullPrescriptionsList([]);
     setPdfTxtGenerateStatus({
       status: PDF_TXT_GENERATE_STATUS.InProgress,
       format,
     });
-    await Promise.allSettled([
-      getPrescriptionSortedList(
-        rxListSortingOptions[selectedSortOption].API_ENDPOINT,
-        true,
-      ).then(response =>
-        setFullPrescriptionsList(
-          response.data.map(rx => {
-            return { ...rx.attributes };
-          }),
-        ),
-      ),
-      !allergies && dispatch(getAllergiesList()),
-    ]);
+    await Promise.allSettled([!allergies && dispatch(getAllergiesList())]);
```

Comment:
I don't think we need Promise.allSettled anymore if there's only one promise

Reply:
@agravell047 I resolved your feedback.
Please let me know if you have any another issues.

Reply:
@gerard-sullivan Do we need to use await still? I would think it was inside of Promise.allSettled before because it was a promise. Is it not actually a promise?

Reply:
@agravell047 This is what I did.
![image](https://github.com/department-of-veterans-affairs/vets-website/assets/132001256/de80dc37-c3f4-4ae0-9103-8babb2b87b26)

Reply:
@gerard-sullivan I would think if this was inside of Promise.allSettled, it would be: `if (!allergies) await dispatch(getAllergiesList())`. 

This might not be necessary because there are examples in the code that do what you're saying. I just wanted to point it out in case we wanted to wait for this to finish. Like for example, to accurately show a loading indicator.